src/app/examples/crud/components/crud.component.html

```diff
@@ -1,89 +1,96 @@
+              <mat-icon fontSet="fas" fontIcon="fa-edit" (click)="edit()"
+                        [matTooltip]="'anms.examples.crud.tooltip.edit' | translate"
+                        matTooltipPosition="above">
+              </mat-icon>
             </button>
-          </span>
-          <ng-template #noBook>
-            <span>
-              <button mat-icon-button color="accent" (click)="isEditing = !isEditing">
-                <mat-icon fontSet="fas" fontIcon="fa-eye" *ngIf="isEditing"></mat-icon>
-                <mat-icon fontSet="fas" fontIcon="fa-edit" *ngIf="!isEditing"></mat-icon>
-              </button>
-            </span>
-          </ng-template>
-          <span *ngIf="selectedBook | async as book">
-            <button mat-icon-button color="warn" (click)="delete(book.id, bookForm)">
-              <mat-icon fontSet="fas" fontIcon="fa-trash"></mat-icon>
+            <button mat-icon-button color="warn" (click)="delete()">
+              <mat-icon fontSet="fas" fontIcon="fa-trash"
+                        [matTooltip]="'anms.examples.crud.tooltip.delete' | translate"
+                        matTooltipPosition="above">
+              </mat-icon>
+            </button>
+            <button mat-icon-button (click)="deselect()">
+              <mat-icon fontSet="fas" fontIcon="fa-times"
+                        [matTooltip]="'anms.examples.crud.tooltip.deselect' | translate"
+                        matTooltipPosition="above">
+              </mat-icon>
             </button>
           </span>
         </span>
       </span>
-      <form #bookForm="ngForm" [formGroup]="bookFormGroup" [style.display]="isEditing ? 'inherit' : 'none'">
+
+      <form #bookForm="ngForm" [formGroup]="bookFormGroup" [style.display]="isEditing ? 'block' : 'none'" [ngClass]="routeAnimationsElements">
         <div class="row">
-          <mat-form-field class="col" [ngClass]="routeAnimationsElements">
+          <mat-form-field class="col">
             <input matInput placeholder="{{ 'anms.examples.crud.placeholder1' | translate }}" formControlName="title"
               required autofocus>
             <mat-error *ngIf="bookFormGroup.get('title').invalid">{{ 'anms.examples.crud.placeholder1' | translate }}
               {{ 'anms.examples.crud.error' | translate }}</mat-error>
           </mat-form-field>
         </div>
         <div class="row">
-          <mat-form-field class="col" [ngClass]="routeAnimationsElements">
+          <mat-form-field class="col">
             <input matInput placeholder="{{ 'anms.examples.crud.placeholder2' | translate }}" formControlName="author"
               required>
             <mat-error *ngIf="bookFormGroup.get('author').hasError('required')">{{ 'anms.examples.crud.placeholder2' | translate }}
               {{ 'anms.examples.crud.error' | translate }}</mat-error>
           </mat-form-field>
         </div>
         <div class="row">
-          <mat-form-field class="col" [ngClass]="routeAnimationsElements">
```

Comment:
These classes only makes sense on top level DOM elements which are visible when the page is loaded